docs/_plugins/search_index.rb

```diff
@@ -74,6 +74,11 @@ def download_link
     m ? m[1] : nil
   end
 
+  def model_type
```

Comment:
We need only two possible results here: `model` or `pipeline`.

Reply:
what if the docs, don't have the Type ?

Reply:
Updated